public/app/core/navigation/kiosk.ts

```diff
@@ -1,33 +1,9 @@
-import { t } from '@lingui/macro';
-
-import { AppEvents, UrlQueryMap } from '@grafana/data';
-import { locationService } from '@grafana/runtime';
+import { UrlQueryMap } from '@grafana/data';
 
 import { KioskMode } from '../../types';
-import appEvents from '../app_events';
-
-export function toggleKioskMode() {
-  let kiosk = locationService.getSearchObject().kiosk;
-
-  switch (kiosk) {
-    case 'tv':
-      kiosk = true;
-      appEvents.emit(AppEvents.alertSuccess, [
```

Comment:
i guess it was intentional to remove this? i'm fine with removing the alert, since it vanished after a short period of time anyway, but i do think we should show something to let a user know how to get out of kiosk mode.

maybe some sort of persistent toast message? it needs to be as unintrusive as possible but still give **some** indication to a user how to get out.

Reply:
It is still there but this is handled in AppChromeService

Reply:
@torkelo huh, then maybe there's a bug because i don't see the alert when going into kiosk mode:

https://user-images.githubusercontent.com/20999846/190387907-7567ccc2-8931-48f5-8674-6f79de16926b.mp4

Reply:
ah, I see the bug now!